nixos/release.nix

```diff
@@ -258,6 +258,7 @@ in rec {
   tests.firefox = callTest tests/firefox.nix {};
   tests.firewall = callTest tests/firewall.nix {};
   tests.fleet = callTestOnTheseSystems ["x86_64-linux"] tests/fleet.nix {};
+  tests.fwupd = callTest tests/fwupd.nix {};
```

Comment:
This is a problem on Hydra.  It causes build during evaluation, most likely due to the code around `listFiles`.  It's probably the cause why it doesn't have resources to evaluate trunk-combined ATM.  I disabled the test for now.

Reply:
There's an issue already :-)  https://github.com/NixOS/nixpkgs/issues/32142

Reply:
Right, sorry about that. Still the same old https://github.com/NixOS/nixpkgs/issues/32142, still not sure how to best fix it.